connection.go

```diff
@@ -149,6 +160,8 @@ type Connection struct {
 	lastStreamId uint64
 
 	serverProtocolInfo ProtocolInfo
+	// watchMap is a map of key -> watchSharedData.
```

Comment:
Ctrl+Shift+F on `watchSharedData` finds the only one line: this one.

Reply:
Fixed.